src/plugins/intel_gpu/src/kernel_selector/kernels/sparse_fill_empty_rows/sparse_fill_empty_rows_kernel_ref.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2018-2025 Intel Corporation
```

Comment:
It is advisable to switch operation to the new OCL infrastructure (examples src/plugins/intel_gpu/src/graph/impls/ocl_v2/)
But this can be done in a separate PR